tests/tests_ovirt_elasticsearch.yml

```diff
@@ -425,6 +425,9 @@
         - "retryfailures=.off."
       changed_when: false
 
+    - name: debug
```

Comment:
Second

Reply:
yes - this is a WIP - not ready to merge - trying to understand why this is failing on el7